.github/workflows/test.yml

```
@@ -15,37 +15,113 @@ on:
 
 jobs:
   build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os:
-          - ubuntu-latest
-          # TODO: Fix cross-env issues in Jest before enabling
-          # - windows-latest
+    name: ⚙️ Build
+    runs-on: ubuntu-latest
     steps:
-      # https://github.com/actions/virtual-environments/issues/1187
-      - name: prep windows network
-        run: Disable-NetAdapterChecksumOffload -Name * -TcpIPv4 -UdpIPv4 -TcpIPv6 -UdpIPv6
-        if: ${{ matrix.settings.host == 'windows-latest' }}
-      - uses: actions/checkout@v2
+      - name: 🛑 Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
 
-      - run: echo "::set-output name=version::$(cat .nvmrc)"
+      - name: ⬇️ Checkout repo
+        uses: actions/checkout@v3
+
+      - name: 🥏 get node in .nvmrc
+        run: echo "::set-output name=version::$(cat .nvmrc)"
         id: nvmrc
 
-      - name: Setup node
+      - name: ⎔ Setup node
         uses: actions/setup-node@v3
         with:
           node-version: "${{ steps.nvmrc.outputs.version }}"
           cache: "yarn"
```

This can be simplified to (see #2461)

```suggestion
      - name: ⎔ Setup node
        uses: actions/setup-node@v3
        with:
          node-version-file: ".nvmrc"
          cache: "yarn"
```